resources/schema.py

```diff
@@ -170,6 +170,7 @@ class BerthNode(DjangoObjectType):
         "leases.schema.BerthLeaseNode",
         description="**Requires permissions** to query this field.",
     )
+    is_accessible = graphene.Boolean()
```

Comment:
I think it's one of those fields that graphene could generate itself, if you put it in `Meta.fields.`

Reply:
I tried that, but it was failing:
```
graphql.error.base.GraphQLError: Cannot return null for non-nullable field BerthNode.isAccessible.
```

so adding it explicitly removed the `non-nullable`-ness. But it's a strange situation, I also think just adding it to the `fields=[]` should be enough.

Reply:
ah, ok, probably a bug in graphene where it does not recognize nullable boolean fields 😅